packages/components/src/Tooltip/src/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from "./Tooltip.tsx";
```

Comment:
Do we need to also expose Tooltip index.ts in the global one?

Reply:
oh shoot! good catch. I knew I was forgetting something haha